lib/hostip.c

```diff
@@ -1107,7 +1107,7 @@ CURLcode Curl_loadhostpairs(struct Curl_easy *data)
         infof(data, "Couldn't parse CURLOPT_RESOLVE entry '%s'!\n",
```

Comment:
I think you should change this to a failf() too now when it errors on this condition.